lib/capistrano/slack_notify.rb

```diff
@@ -91,10 +91,11 @@ def revision
     end
 
     def deploy_target
-      [
+      r = revision
+      t = [
         slack_app_name,
         branch
-      ].join('/') + " (#{revision[0..5]})"
+      ].join('/') + (r ? " (#{r[0..7]})" : '')
```

Comment:
one-letter variable names I really don't like. Maybe memoize `revision` so it doesn't need to shell out twice and just use that method?

```ruby
def deploy_target
  [slack_app_name, branch].join('/') + (revision ? " (#{revision[0..7]})" : "")
end
```

Also, why'd you choose 7 instead of 5?